lizmap/modules/lizmap/lib/App/XmlTools.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * app context informations inside a Jelix environment.
+ *
+ * @author    3liz
+ * @copyright 2020 3liz
+ *
+ * @see      https://3liz.com
+ *
+ * @license Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+
+namespace Lizmap\App;
+
+class XmlTools
+{
+    /*
+     * Interprets a string of XML into an object
+     *
+     * @param $xml_str a well-formed XML string
```

Comment:
`string` is missing. The good syntax is `@param string $xml_str  a well-formed XML string`